guides/lagom-dcos-marathon-deploy-microservices/src/main/paradox/index.md

```diff
@@ -0,0 +1,344 @@
+# Deploying Lagom Microservices on DC/OS
+
+<style type="text/css">
+  pre.code-bash::before {
+    content: '$ ';
+    color: #009900;
+    font-weight: bold;
+  }
+</style>
+
+[Lagom](http://www.lagomframework.com/) is an opinionated microservices framework that makes it quick and easy to
```

Comment:
Maybe Lagom is a flexible microservices framework? :wink:

Reply:
👍